domain/controllernode/service/service.go

```diff
@@ -41,15 +42,20 @@ func (s *Service) UpdateDqliteNode(ctx context.Context, controllerID string, nod
 	return errors.Annotatef(err, "updating Dqlite node details for %q", controllerID)
 }
 
-// IsModelKnownToController returns true if the input
-// model UUID is one managed by this controller.
-func (s *Service) IsModelKnownToController(ctx context.Context, modelUUID string) (bool, error) {
-	uuid, err := s.st.SelectModelUUID(ctx, modelUUID)
+// IsKnownDatabaseNamespace reports if the namespace is known to the controller.
+// If the namespace is not valid an error satisfying [errors.NotValid] is
+// returned.
+func (s *Service) IsKnownDatabaseNamespace(ctx context.Context, namespace string) (bool, error) {
+	if namespace == "" {
+		return false, fmt.Errorf("namespace %q is %w, cannot be empty", namespace, errors.NotValid)
+	}
+
+	ns, err := s.st.SelectDatabaseNamespace(ctx, namespace)
```

Comment:
We've consistently been using `GetFoo` ie the verb is `Get` when querying something that returns a single value. We really should use consistent naming. eg we have `GetModelType` in just this PR, plus all the places elsewhere.

Reply:
I was just copying what we had. It was a name change. I'll update it now.

Reply:
Actually this is the pattern set out for Joe in the naming. I am inclined to leave it.